crates/nu-std/std/mod.nu

```diff
@@ -335,3 +335,16 @@ export def repeat [
 
     1..$n | each { $item }
 }
+
+# return a null device file.
+#
+# # Examples
+#     run a command and ignore it's stderr output
+#     > cat xxx.txt e> (null-device)
+export def null-device []: any -> string {
+    if ((sys).host.name | str downcase) == "windows" {
+        "NUL"
```

Comment:
should that be `NUL:` as in the [stdlib candidate addition](https://github.com/nushell/nu_scripts/pull/649/files#diff-92aa723c230c96a675a0e1deacfb29994e5efce3d6909713d7c0f3d9a84681d5R6)?
i thought thre was a reason for that

Reply:
Not to bikeshed too much but I think it'd be better to use the full path `\\.\NUL`. While a plain `NUL` *should* normally get redirected to the proper path, I wouldn't want to rely on that in all contexts.

Reply:
good to know :pray: 

then the annotation might become `-> path` to reflect the change :+1:

Reply:
Thanks!  It's good to know the `\\.\NUL`, and I verified it works on my windows machine